erpnext/hr/doctype/employee_loan/test_employee_loan.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# See license.txt
+from __future__ import unicode_literals
+
+import frappe
+import erpnext
+import unittest
+from frappe.utils import nowdate
+from erpnext.hr.doctype.salary_structure.test_salary_structure import make_employee
+
+class TestEmployeeLoan(unittest.TestCase):
+	def setUp(self):
+		self.create_loan_type()
+		self.employee = make_employee("robert_loan@loan.com")
+		self.create_employee_loan()
+
+	def create_loan_type(self):
+		if not frappe.db.get_value("Loan Type", "Personal Loan"):
+			frappe.get_doc({
+				"doctype": "Loan Type",
+				"loan_name": "Personal Loan",
+				"maximum_loan_amount": 500000,
+				"rate_of_interest": 8.4
+			}).insert()
+	
+	def	create_employee_loan(self):
+		if not frappe.db.get_value("Employee Loan", {"employee":self.employee}, "name"):
+			employee_loan = frappe.new_doc("Employee Loan")
+			employee_loan.update({
+					"employee": self.employee,
+					"company": erpnext.get_default_company(),
+					"posting_date": nowdate(),
+					"loan_type": "Personal Loan",
+					"rate_of_interest": 8.4,
```

Comment:
Don't pass all these values, instead in `validate` of employee_loan, set those values if missing.